code/modules/clothing/spacesuits/rig/rig.dm

```diff
@@ -357,13 +365,19 @@
 
 	set_vision(!offline)
 	if(offline)
+		crushing = FALSE
 		if(offline == 1)
 			for(var/obj/item/rig_module/module in installed_modules)
 				module.deactivate()
 			offline = 2
 			slowdown = offline_slowdown
 		return
 
+	if(crushing)
+		wearer.apply_damage(10) // Applies 10 brute damage to a random extremity each process
+		if(wearer.stat == DEAD)

```

Comment:
This should set the `crushing` var to false. Otherwise, it'll play ad-infinitum.